config/manager/kustomization.yaml

```
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: docker.io/pavolloffay/opentelemetry-operator
-  newTag: dev-e6d96f00-1755261473
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
+  newTag: 0.131.0-36-ga732ae06
```

```suggestion
  newTag: 0.131
```

Oh I guess that should be correct.